test/cpp/api/CMakeLists.txt

```diff
@@ -30,6 +31,7 @@ if (USE_CUDA)
 endif()
 
 add_executable(test_api ${TORCH_API_TEST_SOURCES})
+target_include_directories(test_api PRIVATE ${TH_CPU_INCLUDE})
```

Comment:
Not for this PR, but we should fix this cmake so that we're not slapping these private INCLUDE variables on here